### PR TITLE
feat: add option to skip missing interpreters

### PR DESCRIPTION
**Pull request by P403n1x87**
Added an option to allow riot to ignore any missing requested Python
interpreters.

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -40,6 +40,21 @@ def without_riotfile(
         yield
 
 
+def assert_args(args):
```
[the remark this answers is not in the document]
Comment (P403n1x87) [reply tests/test_cli.py:43]: PR #117

**Comment by brettlangdon** [comment]
While this implementation looks ok, my hesitation is whether we should allow a switch like this to skip missing.

We've talked in the past about not failing to `riot list` if an interpreter is missing, but we should definitely fail hard if you try to `riot run`/`riot generate`/etc and an interpreter is missing.

Just whether we want to go with a single switch like yours that will ignore any missing, or we push people to be more explicit (e.g. only select the versions you have/want to run with).

@Kyle-Verhoog, @majorgreys, or @jd any thoughts on this?

**Comment by P403n1x87** [comment]
> We've talked in the past about not failing to `riot list` if an interpreter is missing, but we should definitely fail hard if you try to `riot run`/`riot generate`/etc and an interpreter is missing.

I totally agree on this. My thought was that riot already fails hard on missing interpreters by default. The switch is there for when one needs to run riot locally and not all the interpreters are available. This way you'd still be able to run tests against what's available on the host

**Comment by jd** [comment]
I think it's fine to have a global like this for now. It's not a switch you'd use in a CI, but for a user it's fine.
We can still add a finer-grained switch if that is ever needed.

**Comment by jd** [comment]
`Changelog / Validate changelog (pull_request) ` is not required in branch protection we might want to add this @brettlangdon 

@P403n1x87 you might want to add a changelog entry for this 😄

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**